exercises/all-your-base/example.rb

```diff
@@ -0,0 +1,32 @@
+module AllYourBase
+  VERSION = 1
+end
+
+class BaseConverter
+  def convert(base_from, number_array, base_to)
+    return if number_array.any?{|number| number < 0 || number >= base_from}
+    return if base_from <= 1 || base_to <= 1
+    return [] unless number_array.any?
+    number_in_base_ten = convert_to_base_ten(number_array, base_from)
+    convert_from_base_10(number_in_base_ten, base_to)
+  end
+
+  def convert_to_base_ten(number_array, base)
```

Comment:
Actually, I'm going from base "a" to base "b" through base 10, that's why the names of the methods are those ones. I'm sure there is a better solution without doing this. I'll see if I can find it and change the example.